mailpoet/tests/integration/Newsletter/Sending/ScheduledTasksRepositoryTest.php

```diff
@@ -63,6 +69,28 @@ public function testItCanGetFutureScheduledTasks() {
     $this->assertSame($expectedResult, $tasks);
   }
 
+  public function testItTouchAllScheduledTasksByIds(): void {
+    $originalUpdatedAt = CarbonImmutable::now()->subDay();
+    $touched[] = $this->scheduledTaskFactory->create('test', null, Carbon::now()->subDay(), null, $originalUpdatedAt);
+    $untouched[] = $this->scheduledTaskFactory->create('test', null, Carbon::now()->subDay(), null, $originalUpdatedAt);
+    $this->repository->touchAllByIds([$touched[0]->getId()]);
+
+    // check touched task
+    $now = CarbonImmutable::createFromTimestamp((int)$this->wp->currentTime('timestamp'));
+    $this->entityManager->clear();
+    $scheduledTask = $this->repository->findOneById($touched[0]->getId());
+    $this->assertInstanceOf(ScheduledTaskEntity::class, $scheduledTask);
+    $updatedAt = $scheduledTask->getUpdatedAt();
+    $this->assertInstanceOf(\DateTime::class, $updatedAt);
+    $this->assertEqualsWithDelta($updatedAt->getTimestamp(), $now->getTimestamp(), 1);
```

Comment:
I see potential for flakiness here. If code execution takes more than 1 second, the test will fail. I think a less strict comparison would work just fine: 
`$this->assertGreaterThan($updatedAt->getTimestamp(), $originalUpdatedAt->getTimestamp());`.